topo_processor/metadata/metadata_loaders/metadata_loader_imagery_historic.py

```diff
@@ -13,52 +13,57 @@ class MetadataLoaderImageryHistoric(MetadataLoader):
     is_init = False
     raw_metadata: Dict[str, Dict[str, str]] = {}
 
-    def is_applicable(self, item: Item) -> bool:
-        if item.data_type != DataType.ImageryHistoric:
-            return False
-        if not is_tiff(item.source_path):
-            return False
+    def is_applicable(self, asset: Asset) -> bool:
         return True
 
-    async def load_metadata(self, item: Item) -> None:
+    async def load_metadata(self, asset: Asset) -> None:
         if not self.is_init:
             self.read_csv()
 
-        item.source_path_basename = os.path.splitext(os.path.basename(item.source_path))[0]
-        if item.source_path_basename not in self.raw_metadata:
-            raise Exception(f"{item.source_path_basename} cannot be found in the csv.")
-        item_metadata = self.raw_metadata[item.source_path_basename]
+        filename = os.path.splitext(os.path.basename(asset.path))[0]
+
+        if filename not in self.raw_metadata:
+            asset.valid.add_error("Asset not found in CSV file", self.name, "Asset not found in CSV file")
```

Comment:
I think we should make the second text optional.

we only need to record a error if a error is thrown